subprojects/packagefiles/curl/meson.build

```diff
@@ -35,13 +35,15 @@ p = mod.add_project('configure',
                        '--disable-unix-sockets',
                        '--enable-option-checking=fatal',
                        '--enable-websockets',
-                       '--with-openssl',
+                       '--without-openssl',
```

Comment:
I know we spoke about this one, but i think we should make a issue to fix this as i think it will limit us. or, should we introduce a meson option for this as well?

Reply:
this is now #57